pkg/controller/utils/utils.go

```diff
@@ -9,12 +9,17 @@ import (
 	ccaws "github.com/openshift/cloud-credential-operator/pkg/aws"
 
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
 	"github.com/aws/aws-sdk-go/service/iam"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
+const (
+	rootUserDetected = "Detected using root account credentials. Using the AWS account root user is not recommended: https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html"
```

Comment:
nit: I don't think we need "Detected using root account credentials.", because "Using the AWS account root user is not recommended" seems to clearly imply that we think you are (to me anyway).  But I don't feel particularly strongly about it, and I'm not a maintainer over here anyway ;).